logging/logging.go

```diff
@@ -11,29 +11,40 @@ import (
 )
 
 var (
-	VerboseFlag     bool
+	// VerboseFlag enables verbose logging if set to true.
+	VerboseFlag bool
+	// VeryVerboseFlag enables very verbose logging if set to true.
 	VeryVerboseFlag bool
-	Verbose         *log.Logger
-	VeryVerbose     *log.Logger
-	Error           *log.Logger
+	// Verbose is this package's verbose Logger.
+	Verbose *log.Logger
+	// VeryVerbose is this package's very verbose Logger.
+	VeryVerbose *log.Logger
+	// Error is this package's error Logger.
+	Error *log.Logger
 )
 
+// Counter defines an interface for a monotonically incrementing counter.
```

Comment:
users of the Counter API aren't meant to read the value. Counter values are rendered periodically in the logs, and so LogCounter.String() is invoked to display the value.